boilerplate/openshift/osd-container-image/standard.mk

```diff
@@ -0,0 +1,112 @@
+# Validate variables in project.mk exist
+ifndef IMAGE_REGISTRY
+$(error IMAGE_REGISTRY is not set; check project.mk file)
+endif
+ifndef IMAGE_REPOSITORY
+$(error IMAGE_REPOSITORY is not set; check project.mk file)
+endif
+ifndef IMAGE_NAME
+$(error IMAGE_NAME is not set; check project.mk file)
+endif
+ifndef VERSION_MAJOR
+$(error VERSION_MAJOR is not set; check project.mk file)
+endif
+ifndef VERSION_MINOR
+$(error VERSION_MINOR is not set; check project.mk file)
+endif
```

Comment:
This stuff is pretty unnecessary, unless we're trying to enforce the consumer setting a particular variable we're not defaulting (see above).

For that matter, there's really no reason for project.mk to be separated out -- its contents can just be inlined here.